orchestra/proc-macro/src/impl_orchestra.rs

```diff
@@ -128,7 +128,7 @@ pub(crate) fn impl_orchestra_struct(info: &OrchestraInfo) -> proc_macro2::TokenS
 			/// Broadcast a signal to all subsystems.
 			pub async fn broadcast_signal(&mut self, signal: #signal_ty) -> ::std::result::Result<(), #error_ty > {
 				#(
-					let _ = self. #subsystem_name .send_signal(signal.clone()).await;
+					self. #subsystem_name .send_signal(signal.clone()).await?;
```

Comment:
One curious thing in Rust: `_` is a non-binding association, so if the intention was to drop the result after the block it would not be hold...

Reply:
Hey @vstakhov , I don't quite get how that is related? The only reason it exists was to make the compiler :shushing_face: on the unused result.

Reply:
I am not sure what the intention was, I'd speculate this has gone through some refactoring and original reasoning cannot be recovered. Maybe @drahnr might know something

Reply:
This was part of the initial implementation, before it became a framework, before disputes existed, and when the base assumption was that the signal channels are almost always empty and "are not a problem".